### PR TITLE
Update comparison tool filter questions text and online school "learn more" modal

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/13929

## Testing done
Dev tested

## Screenshots
image
image
image
image
image
image
image

## Acceptance criteria
- [x] The "Type of institution" label is replaced with "Select an institution"  on the Landing Page

- [x] The "Select an institution type" label is replaced with "Select an institution" on the Search Results Page

- [x] The "Enter a school, location, or employer name" question label is replaced with "Enter a school, employer name, city, or zip code"
    a. On the Landing Page
    b. On the Search Results Page

- [x] The "How do you want to take classes?" question label is replaced with "Will you be taking any classes in person?"
    a. On the Landing Page
    b. On the Search Results Page
    c. On the Profile Page

- [x] The available answers to "Will you be taking any classes in person?" are: Yes (default selection on page load), No
    a. On the Landing Page
    b. On the Search Results Page
    c. On the Profile Page
    d. Answering "Yes" to "Will you be taking any classes in person?" is equivalent to selecting "In person only" or "In person and online" in the old question
    e. Answering "No" to "Will you be taking any classes in person?" is equivalent to selecting "Online only"

- [x] Will you be taking any classes in person?" has a "(Learn more)" link with the following content:
    a. On the Landing Page
    b. On the Search Results Page
    c. On the Profile Page

    > **Your housing allowance is determined by where you take classes**
    >
    > Under the GI Bill you’re eligible to receive a monthly housing allowance. We calculate this monthly housing allowance based where you take classes.
    >
    > If you use Post-9/11 GI Bill benefits to take at least 1 in-person class, your housing allowance is based on where you do most of your coursework. If you take online courses only, your housing allowance is based on 50% of the national average.
    >
    > Through Dec. 21, 2020, current and new students can receive in-person allowance rates if their school’s approved program changed from in-person to online learning due to COVID-19.
    >
    > [View the current housing allowance payment rates](https://www.benefits.va.gov/gibill/resources/benefits_resources/rate_tables.asp?_ga=2.144591223.39405460.1542131207-1582256389.1508352376).

- [x] The page matches the layout and structure as specified:
    a. In Supporting Artifact # 1 when viewed on a desktop device
    b. In Supporting Artifact # 2 when viewed on a mobile device

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/search/InstitutionSearchForm.jsx [hunk 1]
```diff
@@ -47,9 +47,13 @@ function InstitutionSearchForm({
   }
 
   const header = gibctFilterEnhancement ? 'Refine search' : 'Keywords';
-  const keywordSearchLabel = gibctFilterEnhancement
+  let keywordSearchLabel = gibctFilterEnhancement
     ? 'Enter a school, location, or employer name'
```
Comment (zurbergram) [inline src/applications/gi/components/search/InstitutionSearchForm.jsx:51]: Should this be the line that is updated?

Comment (hughes-dustin) [reply src/applications/gi/components/search/InstitutionSearchForm.jsx:51]: For now, yes. This is one of those situations where we have a prod flag within a feature flag.

Comment (hughes-dustin) [reply src/applications/gi/components/search/InstitutionSearchForm.jsx:51]: I just liked using a separate condition to make it easy when we remove the prod flag.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**